README.md

```diff
@@ -525,54 +520,27 @@ protection as well as free SSL and other useful features.
 
 ### Node.js modules and CLI tools
 
-For the time being we are using GitHub releases for our `npm` _modules_ and CLI
-tools. This means that modules can be installed via
-`npm i Laboratoria/repo-name#tag`, or adding to your `package.json`.
-
-Example as dependency in `package.json`:
-
-```json
-{
-  "dependencies": {
-    "schemas": "Laboratoria/schemas#v1.0.0-alpha.3"
-  }
-}
-```
-
-Example installing the `curriculum-parser` globally:
-
-```sh
-# install current release
-npm i -g Laboratoria/curriculum-parser
-
-# install a specific version
-npm i -g "Laboratoria/curriculum-parser#v2.0.0-alpha.2"
-```
-
-GitHub vs NPM Orgs???
+@lupomontero ??
```

Comment:
@lupomontero I think we should update this part to show the new way to install packages from @Laboratoria

Reply:
Yup. Do you want to give it a go? We need to mention the `@laboratoria` namespace on NPM for public modules and for private repos I would continue to recommend installing from repo. Thoughts?

Reply:
I believe We just have public packages under `@laboratoria` namespace on NPM. But, We probably have private repos on GitHub that we might want to use as samples for "installing from repos" section here. So, I'd prefer to leave to you, you know those repos better.

BTW, we just made the `Laboratoria/models` repo private on GitHub, and the next step, I think, should be merge it with the `Laboratoria/api.laboratoria.la` repo. So, If that step is done,  we might want to make the package `@Laboratoria/models` private on NPM as `Laboratoria/api.laboratoria.la` is private, which will affect other depedent repos where you are the maintainer.  All I mentioned about this next step is already as PR, but I'm not taking part of it, because as you know I'm leaving soon, and I firmly believe making no big changes is better at that stage.

I wrote a lot about a supposition. Back to the your comment, I'd leave to you :sweat_smile:

Reply:
WRT `@laboratoria/models`, this is also a dependency in `@laboratoria/curriculum-parser`, which is a public package/repo, as it iself is a dependency in `Laboratorias/bootcamp`, which HAS to be a public repo... :thinking:

Reply:
Para no bloquear este PR me he tomado la libertad de mover este tema a un issue aparte. Ver #34